setup.cfg

```diff
@@ -73,14 +73,15 @@ install_requires =
     cookiecutter >= 1.7.3  # dependency issues in older versions
     dataclasses; python_version<"3.7"
     enrich >= 1.2.5
+    importlib-metadata<2; python_version<="3.7"
```

Comment:
Why isn't this listed among constraints?

Reply:
That is because pip-tools bugs/limitations. I am considering removing the constraints soon unless someone has time to split them python version. Even if we split them per python versions, we will also need one for linux and non linux.

Fixing the deps/constraints is not as easy as it seams. Happy to discuss that but is outside the scope of that PR.

Reply:
This appears to cause problems when trying to install ansible-core + molecule + an openstack component (ex. os-net-config)  via constraints when installing under centos8 (e.g. 3.6).  importlib-metadata 4.8 works, what is the specific reason to lock it to 2?